src/Misc/layoutbin/systemd.svc.sh.template

```diff
@@ -106,15 +106,27 @@ function stop()
 
 function uninstall()
 {
-    stop
-    systemctl disable ${SVC_NAME} || failed "failed to disable  ${SVC_NAME}"
-    rm "${UNIT_PATH}" || failed "failed to delete ${UNIT_PATH}"
+    if service_exists; then
+        stop
+        systemctl disable ${SVC_NAME} || failed "failed to disable ${SVC_NAME}"
+        rm "${UNIT_PATH}" || failed "failed to delete ${UNIT_PATH}"
+    else
+        echo "Service ${SVC_NAME} is not installed"
+    fi    
     if [ -f "${CONFIG_PATH}" ]; then
-      rm "${CONFIG_PATH}" || failed "failed to delete ${CONFIG_PATH}"
+        rm "${CONFIG_PATH}" || failed "failed to delete ${CONFIG_PATH}"
```

Comment:
extra spaces. 😄

Reply:
Ah, I thought a 4 space indent is standard so I changed it on purpose :)
Should we stick to 2 space in `.sh` files?

Reply:
I think we don't change the format in this PR. If you want, we can change the format for the entire file later.